### ISSUE TITLE
NOT A ISSUE

### ISSUE THREAD

**Issue by apifarmer** [Bunsly/HomeHarvest#91]
this is very cool. i wanted to thank you for the time and dedication you put into this. could i ask the general routing of how this works? i've tried to do this with the async-page-create and could not figure it out.

**Comment by ZacharyHampton** [comment]
async-page-create is from Zillow, no? We use realtor.com

**Comment by apifarmer** [comment]
yeah it is i figured you had insight since i seen somewhere  (maybe a old reddit post) this could scrape all 3 zillow refin and realtor, but i see its realtor now that im using it.

**Comment by apifarmer** [comment]
are there more in-depth docs?

**Comment by ZacharyHampton** [comment]
The readme should have everything you need to know. If not, let me know and I can answer them here.

**Comment by apifarmer** [comment]
i've tried to run it local with no luck only if i do in your video..  is this missing a module or function? why is it listed as mit license if you're not going to post the functional code..

**Comment by apifarmer** [comment]
![image](https://github.com/user-attachments/assets/d6689e53-e158-4666-a11b-35a347fbe356)

**Comment by apifarmer** [comment]
are the files uploaded not the same as the module being pip installed? my code has this above

**Comment by ZacharyHampton** [comment]
Everything required is in the repo. If you are looking for Zillow code, that's been removed for a while now. You must go back to old commits so see such code.

**Comment by apifarmer** [comment]
Is the zillow code functional or shows a good approach?

**Comment by ZacharyHampton** [comment]
It used to be functional, who knows if anything changed and if it no longer works.
